templates/partials/actionbar.html

```diff
@@ -44,19 +44,20 @@
 
       <div class="col-sm-4 general-actions left-pane" ng-show="currentTab === 'contacts'">
         <div class="actions dropup">
-          <a class="mm-icon mm-icon-inverse mm-icon-caption" ui-sref="contacts.add()" ng-show="facilities.length">
+          <a class="mm-icon mm-icon-inverse mm-icon-caption" ng-show="actionBar.left.userFacilityId" ui-sref="contacts.addChild({ type: actionBar.left.userChildPlace.type, parent_id: actionBar.left.userFacilityId })">
             <span class="fa-stack">
-              <i class="fa fa-building fa-stack-1x"></i>
+              <i class="fa {{actionBar.left.userChildPlace.icon}} fa-stack-1x"></i>
               <i class="fa fa-plus fa-stack-1x"></i>
             </span>
-            <p translate>Add place</p>
+            <p translate>action.{{actionBar.left.userChildPlace.type}}.add</p>
           </a>
-          <a class="mm-icon mm-icon-inverse mm-icon-caption" ui-sref="contacts.add()" ng-show="!facilities.length">
+
+          <a class="mm-icon mm-icon-inverse mm-icon-caption" ui-sref="contacts.add()" ng-show="!actionBar.left.userFacilityId">
             <span class="fa-stack">
-              <i class="fa fa-group fa-stack-1x"></i>
+              <i class="fa fa-building fa-stack-1x"></i>
               <i class="fa fa-plus fa-stack-1x"></i>
             </span>
-            <p translate>action.clinic.add</p>
+            <p translate>Add place</p>
```

Comment:
Oh, I see. Good point.
Turns out it is a translation key, from before they were standardized.

On Mon, Sep 5, 2016 at 2:22 PM, Stefan du Fresne <notifications@github.com>
wrote:

> In templates/partials/actionbar.html
> <https://github.com/medic/medic-webapp/pull/2708#discussion_r77514029>:
>
> >                <i class="fa fa-plus fa-stack-1x"></i>
> >              </span>
> > -            <p translate>action.clinic.add</p>
> > +            <p translate>Add place</p>
>
> I meant that actual "Add place" bit and it not being a translation key.
> Not sure what our standard is for this kind of thing, I presumed we used
> keys?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/medic/medic-webapp/pull/2708/files/de5ce9c2c3ad588df94d896304677bb15ef48b7a#r77514029>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA3oShLzA1tkEYG9TLRGhTXsj6e11Uc6ks5qnAmNgaJpZM4J06dk>
> .
>